app/javascript/analytics/dashboard.js

```diff
@@ -165,48 +165,65 @@ function drawCharts(data, timeRangeLabel) {
   });
 }
 
-function callAnalyticsApi(date, timeRangeLabel, { organizationId, articleId }) {
-  let url = `/api/analytics/historical?start=${
-    date.toISOString().split('T')[0]
-  }`;
-
-  if (organizationId) {
-    url = `${url}&organization_id=${organizationId}`;
-  }
-  if (articleId) {
-    url = `${url}&article_id=${articleId}`;
+function renderReferrers(data) {
+  const container = document.getElementById('referrers-container');
+  const tableBody = data.domains
+    .filter(referrer => referrer.domain)
+    .map(referrer => {
+      return `
+      <tr>
+        <td>${referrer.domain}</td>
+        <td>${referrer.count}</td>
+      </tr>
+    `;
+    });
+
+  // add referrers with empty domains if present
```

Comment:
Some page views don't have a referrer, hence they won't have a domain. We append this at the top of the table